subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -143,17 +147,39 @@ uint8_t ll_setup_iso_path(uint16_t handle, uint8_t path_dir, uint8_t path_id,
 	dp->coding_format = coding_format;
 	dp->company_id    = company_id;
 
-	/* TBD dp->sync_delay    = controller_delay; ?*/
+	/* TODO dp->sync_delay    = controller_delay; ?*/
+
+	cig = cis->group;
```

Comment:
This CIS-specific code needs to be under compilation guards. I am a bit surprised that line 110 compiles and links in BIS-only builds.

Reply:
Fair point, now uses guards